tests/Unit/Subscriber/Cache/PurgeActionsSubscriber/TestPurgeUserCache.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace WP_Rocket\Tests\Unit\Subscriber\Cache\PurgeActionsSubscriber;
+
+use WP_Rocket\Subscriber\Cache\PurgeActionsSubscriber;
+use WP_Rocket\Tests\Unit\TestCase;
+use Brain\Monkey\Functions;
+use Brain\Monkey\Filters;
+
+/**
+ * @coversDefaultClass \WP_Rocket\Subscriber\Cache\PurgeActionsSubscriber
+ */
+class TestPurgeUserCache extends TestCase {
+	/**
+	 * @covers::purge_user_cache
+	 * @group purge_actions
+	 */
+	public function testShouldReturnNullWhenUserCacheDisabled() {
```

Comment:
The method always returns `null` no matter what happens. I’d suggest renaming the test to something like: `testShouldNotPurgeUserCacheWhenUserCacheDisabled`. With this naming technique, it tells us what the expected behavior is when something happens.